### ISSUE TITLE
Martingale tidak sesuai perintah.......Mohon dibantu

### ISSUE THREAD

**Issue by Yusriadi78** [Fruitfella/ROBOT-binary#261]
Jika Para mastah yang punya waktu luang, tolong di bantu memperbaiki fungsi martingale dan compound di BOT ini.......

saya punya masalah pada fungsi BOT ini....
1. Jika Win di level compound ke 2 dan Loss di Level Compound ke 3, BOT langsung martingale yang loss level 3.
2. Saya mau martingale di Initial stake list ke 2

Begitu seterusnya sampai mencapai level martingale (stake list)

![mohon bantuan](https://user-images.githubusercontent.com/38336159/39618593-a3c9bf18-4fae-11e8-9840-ad5481fb2b2b.png)

[MOHON BANTUAN.txt](https://github.com/Fruitfella/ROBOT-binary/files/1973656/MOHON.BANTUAN.txt)

Terima kasih bila ada meluangkan waktunya untuk membantu....

@AlienFly  
@valentino242  
@aminmarashi  
@ubanemas  
@okovalov  
@Fruitfella  
@gokula-krishna-dev  
@binary-ex-machina

**Comment by valentino242** [comment]
Kurang detail permintaannya.....  

Compound ke 3 loss = Martingale stake list 2 , loss lagi ke stake list 3 dst?
Kalau Profit level compoundnya balik ke 1 lagi ? atau terus berdasarkan contract detail profitnya ?

**Comment by valentino242** [comment]
Coba kasih penjelasan yang detail . Barangkali saya bisa bantu

**Comment by Yusriadi78** [comment]
Terimakasih responnya bro @valentino242 .....

1. Stake Awal $1 di Compound 4x tanpa loss dan kembali ke Stake Awal....
2. Jika Loss di Compound level 2/3 maka Win terhitung 0 dan berlanjut ke Initial Stake List No. 2.....Jika Memenuhi 4x Compound maka kembali ke Stake Awal....

**Comment by Yusriadi78** [comment]
Pada Gambar diatas masih ada Marti Factor dan saya tidak mau pake itu tapi pake aturan Initial Stake List....

**Comment by valentino242** [comment]
1. Stake Awal $1 di Compound 4x tanpa loss dan kembali ke Stake Awal. ( Win Counter = 0)
2. Setelah kembali ke stake awal lagi  dan jika Win Counter =2 dan Loss Counter = 1 , Maka Martingale sesuai stake list sampai profit? 
3. Jika Profit maka kembali ke stake awal $1 dan  Win Counter = 0 ?

**Comment by Yusriadi78** [comment]
@valentino242

1. Sudah benar
2. Setelah kembali ke stake awal lagi dan jika Win Counter =2 dan Loss Counter = 1 , Maka Martingale sesuai stake list 
3. Jadi (Initial Stake step berubah menjadi = 2) & di Compound 4x tanpa Loss dan kembali ke stake awal $1 Jika memenuhi 4x Compound

**Comment by Yusriadi78** [comment]
![untitled](https://user-images.githubusercontent.com/38336159/39659955-eced85a2-505e-11e8-9d8f-28c9108fda2c.png)


1. Initial Stake Step 1 (Modal $1) di Compound 4x
 - Jika Initial Stake Step 1 terjadi Win Counter = 2 & Loss Counter = 1, Maka Win Counter = 0 (Lanjut ke Initial Stake Step 2 (Modal $2)
2. Initial Stake Step 2 (Modal $2) di Compound 4x 
- Jika Initial Stake Step 2 Win Compound 4x Tanpa Loss Maka kembali ke stake awal $1
- Jika Initial Stake Step 2 terjadi Win Counter = 2 & Loss Counter = 1, Maka Win Counter = 0 (Lanjut ke Initial Stake Step 3 (Modal $3)
3. Initial Stake Step 3 (Modal $3) di Compound 4x
- Jika Initial Stake Step 3 Win Compound 4x Tanpa Loss Maka kembali ke stake awal $1
- Jika Initial Stake Step 3 terjadi Win Counter = 2́ atau 3 & Loss Counter = 1, Maka Win Counter = 0 (Lanjut ke Initial Stake Step 4 (Modal $6)
4. Initial Stake Step 4 (Modal $6) di Compound 4x
- Jika Initial Stake Step 4 Win Compound 4x Tanpa Loss Maka kembali ke stake awal $1
- Jika Initial Stake Step 4 terjadi Win Counter = 1 atau2́ atau 3 & Loss Counter = 1, Maka Win Counter = 0 (Lanjut ke Initial Stake Step 5 (Modal $12)

Akan terus berlanjut sampai Initial Stake Step 8 atau Max Level Marti tidak beraturan (Modal $96)

**Comment by Yusriadi78** [comment]
Maaf Bro @valentino242 sudah merepotkan

**Comment by valentino242** [comment]
Oke minta email aja nanti kalau saya senggang saya buatin.

**Comment by Yusriadi78** [comment]
Ini email saya bro (yusriadi78@gmail.Com)
@valentino242

**Comment by Yusriadi78** [comment]
Help me @delar4ik @calordum

**Comment by valentino242** [comment]
Bahas rule awal lagi = Jika Win di level compound ke 2 dan Loss di Level Compound ke 3, BOT langsung martingale yang loss level 3?
Jadi seperti ini : 
STAKE $1 ( RULE1)
OP1 = PROFIT || LANJUT COMPOUND
OP2 =PROFIT || LANJUT COMPOUND
OP3= LOSS || PINDAH KE STAKE LIST $2 MARTINGALE (COMPOUND 4X PROFIT KEMBALI KE STAKE $1) ?

Kalau seperti ini gmn ?
STAKE$1
OP1 = LOSS 
OP2= LOT TETAP? 
OP3= PROFIT ||  LANJUT COMPOUND SAMPAI LEVEL COMPOUND TERPENUHI DAN KEMBALI KE STAKE $1 ?
(Ini tidak akan pindah ke stake$2 kalau tidak tercapai Rule 1) Bener g seperti ini?) Martingale pindah Stake list  hanya terjadi jika Rule 1 terpenuhi?

**Comment by Yusriadi78** [comment]
Trimakasih @valentino242 

Intinya Jika Level Compound Ke 2 dan Loss di Level Compound Ke 3, Maka Lanjut ke Stake List Nomor 2, bukan yang level ketiga di martingale tapi mengambil stake berurutan pada daftar List yang di sediakan....

**Comment by Yusriadi78** [comment]
Harus memenuhi 4x Compound untuk kembali ke Stake Awal

**Comment by valentino242** [comment]
Lah iya matingale khan ambil di stake list !

**Comment by valentino242** [comment]
Compound ambil dari contract detail profit
Martingale ambil dari stake list

**Comment by Yusriadi78** [comment]
@valentino242 

Saya bingung cara nyusun BOT.nya Bro, masih banyak perintah yang kurang paham......

**Comment by valentino242** [comment]
Sorry Ga bisa bantu coz pemahaman saya lemah jadi masih ga mudeng maksudnya . Km sendiri yang nanya ga tau ambil compoundnya seperti apa . Apalagi saya yang mau bantuin. Mungkin Master" yang lain dan Om Suhu fruit bisa bantu. 

Coba kesini 💯 
https://github.com/binary-com/binary-bot/issues/646
